infrastructure/modules/host.nix

```diff
@@ -0,0 +1,101 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+let
+  # This script is run when deployuser logs in via ssh.
+  # Modifications to this script (like changing the branch name), should be manually deployed with
+  # deploy-rs. 'Dangerous' changes to system configuration like modifying users or changing network setting
+  # should also be done with deploy-rs to take advantage of the rollback system.
+  deployuserScript = pkgs.writeShellScriptBin "deployuser-script" ''
+    #!/usr/bin/env bash
+    set -ex
+
+    branch="nixification"
+
+    if [ -d "catcolab" ]; then
+      rm -rf ./catcolab
+    fi
+
+    git clone -b "$branch" https://github.com/ToposInstitute/CatColab.git catcolab
```

Comment:
If this is running in CI, it should take the git hash of the commit that it is running on